src/main/java/com/blackducksoftware/integration/hub/alert/channel/email/EmailMessagingService.java

```diff
@@ -82,12 +77,8 @@ public void sendEmailMessage(final EmailTarget emailTarget) throws MessagingExce
 
         final Session session = createMailSession(emailProperties);
         final Map<String, String> contentIdsToFilePaths = new HashMap<>();
-        if (StringUtils.isNotBlank(emailProperties.getEmailTemplateLogoImage())) {
-            addTemplateImage(model, contentIdsToFilePaths, EmailProperties.EMAIL_LOGO_IMAGE, emailProperties.getEmailTemplateLogoImage());
-        } else {
-            // TODO determine the actual image location for deployment
-            addTemplateImage(model, contentIdsToFilePaths, EmailProperties.EMAIL_LOGO_IMAGE, System.getProperties().getProperty("user.dir") + "/src/main/resources/email/images/Ducky-80.png");
-        }
+
+        addTemplateImage(model, contentIdsToFilePaths, EmailProperties.EMAIL_LOGO_IMAGE, System.getProperties().getProperty("user.dir") + "/src/main/resources/email/images/Ducky-80.png");
```

Comment:
We should leave the TODO message since this location will change in the final release